### PR TITLE
Piotr: Working fetching of random joke from ChuckApi

### PR DESCRIPTION
**Pull request by piogasio**

### CODE REVIEW

File: cnjokes/src/RandomJoke.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            joke: null,
+        };
+    }
+
+    getRandomJoke = () => {
+        fetch(CHUCK_API + RANDOM_JOKE_QUERY)
```
Comment (CorwinCZ) [inline cnjokes/src/RandomJoke.js]:
```js
`${CHUCK_API}${RANDOM_JOKE_QUERY}`
```
Could be better :)

File: cnjokes/src/RandomJoke.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        this.getRandomJoke();
+    }
+
+
+    render() {
+        const { joke } = this.state;
+
+        return (
+            <div id="chuckWrap">
+                <div id="chuckHeader"></div>
```
Comment (CorwinCZ) [inline cnjokes/src/RandomJoke.js]: Please use `class` instead of `id`

Comment (CorwinCZ) [reply cnjokes/src/RandomJoke.js]: Also this div can be self-closing - `<div class="chuckHeader" />`

Comment (piogasio) [reply cnjokes/src/RandomJoke.js]: Done

File: cnjokes/src/RandomJoke.js (hunk 1 above)
Comment (piogasio) [reply cnjokes/src/RandomJoke.js]: Done

**Review by lukasrzavsky — APPROVED** [review]

**Pull request merged**